src/app/page.js

```diff
@@ -128,6 +128,20 @@ export default function Home() {
             </div>
           </div>
         </div>
+        <div className="bg-red-50 border-l-4 border-red-500 p-6 rounded-lg shadow-lg">
```

Comment:
Podemos añadir un pequeño margen para que no se superponga el boton del menú?
```suggestion
        <div className="mt-20 bg-red-50 border-l-4 border-red-500 p-6 rounded-lg shadow-lg">
```